react/react1/week1/my-page/src/TodoItems.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+function TodoItem(props) {
```

Comment:
You might check Object destructuring here, and use it like `{ description, deadline }` and thus you might use it directly. A cool feature.

Reply:
done everything